### PR TITLE
[android] Cleanup debug display

### PR DESCRIPTION
**Pull request by cco3**
This change:
* Changes a few variable names to be more fitting
* Sets all values in the adapter to avoid displaying recycled values
* Reorganizes the metadata debug line so that scan time comes first

### CODE REVIEW

File: android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java [hunk 1]
```diff
@@ -484,10 +484,12 @@ public View getView(int i, View view, ViewGroup viewGroup) {
       PwsClient.UrlMetadata urlMetadata = mUrlToUrlMetadata.get(url);
       // If the metadata exists
       if (urlMetadata != null) {
+        Log.d(TAG, url + " Metadata not null1");
```
Comment (mmocny) [inline android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java]: woops, didn't notice the spurious `1` before landing this.

File: android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java [hunk 2]
```diff
@@ -495,6 +497,7 @@ public View getView(int i, View view, ViewGroup viewGroup) {
       }
       // If metadata does not yet exist
       else {
+        Log.d(TAG, url + " Metadata null1");
```
Comment (mmocny) [inline android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java]: and here.

File: android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java [hunk 3]
```diff
@@ -545,25 +549,31 @@ private void updateRangingDebugView(String url, View view) {
...
+      float scanTime = mUrlToScanTime.get(url) / 1000.0f;
+      String scanTimeString = getString(R.string.metadata_debug_scan_time_prefix)
+          + new DecimalFormat("##.##s").format(scanTime);
+      TextView scanTimeView = (TextView) view.findViewById(R.id.metadata_debug_scan_time);
+      scanTimeView.setText(scanTimeString);
+
+      TextView rankView = (TextView) view.findViewById(R.id.metadata_debug_rank);
+      TextView pwsTripTimeView = (TextView) view.findViewById(R.id.metadata_debug_pws_trip_time);
       if (metadata != null) {
+        Log.d(TAG, url + " Metadata not null2");
```
Comment (mmocny) [inline android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java]: I'm starting to think these are not accidents :P

File: android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java [hunk 4]
```diff
@@ -484,10 +484,12 @@ public View getView(int i, View view, ViewGroup viewGroup) {
       PwsClient.UrlMetadata urlMetadata = mUrlToUrlMetadata.get(url);
       // If the metadata exists
       if (urlMetadata != null) {
+        Log.d(TAG, url + " Metadata not null1");
         // Set the title text
         titleTextView.setText(urlMetadata.title);
         // Set the url text
-        urlTextView.setText(urlMetadata.displayUrl);
```
Comment (g-ortuno) [inline android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java]: Why did you change it back to `url`?

**Comment by cco3** [comment]
All debugging reasons, sorry...I had a TODO to fix it but accidentally removed it.
